101-app-service-regional-vnet-integration/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
+            "value": "GEN-UNIQUE"
+        },
+        "appServicePlanName": {
```

Comment:
remove params once you have defaultValues as mentioned above

Reply:
Done.